### ISSUE TITLE
How to bind event on label of axis-x?

### ISSUE THREAD

**Issue by vvkee** [antvis/g2#979]
As shown in the following figure，How to bind event on axis-x where "150" label?
![image](https://user-images.githubusercontent.com/7991147/47502919-818f9000-d89b-11e8-89d6-c490e6c5fff7.png)

**Comment by Leannechn** [comment]
axis-label 事件绑定失效。3.5.1 之后都版本都失效了，是删除了这个事件还是bug？
chart.on('axis-label:click')

**Comment by dxq613** [comment]
fixed on 3.5.8
```js
chart.on('axis-label:click', function(ev) {
  console.log(ev);
})
```
